porting/building-ubports-boot.rst

```diff
@@ -1,98 +0,0 @@
-Building ubports-boot
-=====================
-
-Ubuntu Touch uses Upstart rather than Systemd for its init daemon. Because of this, it is not fully Halium-compatible and is not able to use the vanilla hybris-boot that Halium produces. For this reason, we need to build ubports-boot.
-
-
-Add source to Halium tree
--------------------------
-
-The first thing that needs to be done is adding the ubports-boot source to your Halium tree. You may choose to do this by adding it to your local manifests (recommended) or simply cloning it in place.
-
-
-Add to local manifest
-^^^^^^^^^^^^^^^^^^^^^
-
-Create a new file in ``.repo/local_manifests`` called ``ubports-boot.xml``. Add the following to the file:
-
```

Comment:
We need something similar to this, otherwise you don't get the halium-boot repo right now. 

Probably good to add a TOOD along with it to remove it later if halium-boot does get merged in halium.

Reply:
I'm way ahead of you: https://github.com/Halium/android/pull/23 https://github.com/Halium/android_build/pull/14